formats/json/commonTest/src/kotlinx/serialization/SerializersLookupTest.kt

```diff
@@ -207,6 +205,29 @@ class SerializersLookupTest : JsonTestBase() {
         assertEquals("42", json.encodeToString(42))
     }
 
+    class NonSerializable
+
+    class NonSerializableBox<T>(val boxed: T)
+
+    @Test
+    fun testLookupFail() {
```

Comment:
It's also good to test that this function works for serializable classes :)

Reply:
That's what the rest of test do, doesn't it?